libnetwork/resolvconf/resolvconf.go

```diff
@@ -1,22 +1,16 @@
 // Package resolvconf provides utility code to query and update DNS configuration in /etc/resolv.conf
+//
+// Its functions are implemented in terms of 'libnetwork/internal/resolvconf', which is specialised
+// for use by libnetwork's use-cases.
 package resolvconf
 
 import (
 	"bytes"
-	"context"
+	"fmt"
 	"os"
-	"regexp"
 	"strings"
-	"sync"
-
-	"github.com/containerd/log"
-)
 
-const (
-	// defaultPath is the default path to the resolv.conf that contains information to resolve DNS. See Path().
-	defaultPath = "/etc/resolv.conf"
-	// alternatePath is a path different from defaultPath, that may be used to resolve DNS. See Path().
-	alternatePath = "/run/systemd/resolve/resolv.conf"
+	newrc "github.com/docker/docker/libnetwork/internal/resolvconf"
```

Comment:
[Avoid renaming imports except to avoid a name collision.](https://go.dev/wiki/CodeReviewComments#imports) There is nothing else named `resolvconf` in this scope.
```suggestion
	"github.com/docker/docker/libnetwork/internal/resolvconf"
```

Reply:
Ok, I thought referring to (other) `resolvconf` in `resolvconf` might be confusing. But, done.